release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -58,6 +58,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-9-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-9-aws-china-regions"]
+==== Support for AWS public regions in China
```

Comment:
Since AWS China regions are different from AWS Global regions (like `us-east-2`), I do not think they are _**public regions**_ (like GovCloud), b/c 1) no rhcos images are published in these regions, b) not listed in `openshift-installer`.

Reply:
Thank you. Removed references to `public`.